Update abnormalLevelOfChemicalEntityWithRole.yaml

nicolevasilevsky opened the pull request:
addresses https://github.com/obophenotype/upheno/issues/635

srobb1 commented:
This new example [HP_0030347](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0030347) includes a location, blood. I think the example should be simple like: 
`An abnormal level of a chemical with a specific role, such as a metabolite or hormone like androgen.`


HP_0030347:
```
has part some (
    amount and
    inheres in some (
        chemical entity and
        part of some blood and
        has role some androgen) and
    has modifier some abnormal)
```

nicolevasilevsky commented:
That sounds good to me, @srobb1. I'll update this.

nicolevasilevsky commented:
I removed the specific HPO example, because that was in a location. I don't think HPO has any actual examples that match this pattern, so I just added a hypothetical example.
